examples/simple/package-lock.json

```diff
@@ -0,0 +1,777 @@
+{
```

Comment:
The example won't run as given; it will prompt the user to run npm install, then link the `xyz` package. But since I plan to add the more recent TS template as an example anyway : #38